Examples/Scripts/Python/truth_tracking.py

```diff
@@ -0,0 +1,225 @@
+    vtxGen = acts.examples.GaussianVertexGenerator()
+    vtxGen.stddev = acts.Vector4(0, 0, 0, 0)
+
+    ptclGen = acts.examples.ParametricParticleGenerator(
+        p=(1 * u.GeV, 10 * u.GeV), eta=(-2, 2), numParticles=2
+    )
+
+    g = acts.examples.EventGenerator.Generator()
+    g.multiplicity = acts.examples.FixedMultiplicityGenerator()
+    g.vertex = vtxGen
+    g.particles = ptclGen
+
+    evGen = acts.examples.EventGenerator(
+        level=acts.logging.INFO,
+        generators=[g],
+        outputParticles="particles_input",
+        randomNumbers=rnd,
+    )
+    s.addReader(evGen)
+
+    s.addWriter(
+        acts.examples.RootParticleWriter(
+            level=acts.logging.INFO,
+            inputParticles=evGen.config.outputParticles,
+            filePath=str(outputDir / "particles.root"),
+        )
+    )
+
+    # Selector
+    selector = acts.examples.ParticleSelector(
+        level=acts.logging.INFO,
+        inputParticles=evGen.config.outputParticles,
+        outputParticles="particles_selected",
+    )
+    s.addAlgorithm(selector)
+
+    # Simulation
+    simAlg = acts.examples.FatrasSimulation(
+        level=acts.logging.INFO,
+        inputParticles=selector.config.outputParticles,
+        outputParticlesInitial="particles_initial",
+        outputParticlesFinal="particles_final",
+        outputSimHits="simhits",
+        randomNumbers=rnd,
+        trackingGeometry=trackingGeometry,
+        magneticField=field,
+        generateHitsOnSensitive=True,
+    )
+    s.addAlgorithm(simAlg)
+
+    # Digitization
+    digiCfg = acts.examples.DigitizationConfig(
+        acts.examples.readDigiConfigFromJson(
+            str(digiConfigFile),
+        ),
+        trackingGeometry=trackingGeometry,
+        randomNumbers=rnd,
+        inputSimHits=simAlg.config.outputSimHits,
+    )
+    digiAlg = acts.examples.DigitizationAlgorithm(digiCfg, acts.logging.INFO)
+    s.addAlgorithm(digiAlg)
+
+    selAlg = acts.examples.TruthSeedSelector(
+        level=acts.logging.INFO,
+        ptMin=500 * u.MeV,
+        nHitsMin=9,
+        inputParticles=simAlg.config.outputParticlesInitial,
+        inputMeasurementParticlesMap=digiCfg.outputMeasurementParticlesMap,
+        outputParticles="particles_seed_selected",
+    )
+    s.addAlgorithm(selAlg)
+
+    inputParticles = selAlg.config.outputParticles
+
+    smearAlg = acts.examples.ParticleSmearing(
+        level=acts.logging.INFO,
+        inputParticles=inputParticles,
+        outputTrackParameters="smearedparameters",
+        randomNumbers=rnd,
+        # Gaussian sigmas to smear particle parameters
+        # sigmaD0=20 * u.um,
+        # sigmaD0PtA=30 * u.um,
+        # sigmaD0PtB=0.3 / u.GeV,
+        # sigmaZ0=20 * u.um,
+        # sigmaZ0PtA=30 * u.um,
+        # sigmaZ0PtB=0.3 / u.GeV,
+        # sigmaPhi=1 * u.degree,
+        # sigmaTheta=1 * u.degree,
+        # sigmaPRel=0.01,
+        # sigmaT0=1 * u.ns,
+        # initialVarInflation=[1, 1, 1, 1, 1, 1],
```

Comment:
These should be the default values, I just kept them here as comments to make clear that this can be configured. Can remove if unclear / unnecessary.